docs/_docs/quickstart.md

```diff
@@ -26,7 +29,7 @@ If you encounter any unexpected errors during the above, please refer to the [tr
 
 ## About Bundler
 
-`gem install jekyll bundler` installs the [jekyll](https://rubygems.org/gems/jekyll/) and [bundler](https://rubygems.org/gems/bundler) gems through [RubyGems](https://rubygems.org/). You need only to install the gems one time &mdash; not every time you create a new Jekyll project. Here are some additional details:
+`gem install bundler` installs the [bundler](https://rubygems.org/gems/bundler) gem through [RubyGems](https://rubygems.org/). You need only to install it once &mdash; not every time you create a new Jekyll project. Here are some additional details:
```

Comment:
You may consider rewording this sentence to:
```diff
- You need only to install it once &mdash; not every time you create a new Jekyll project.
+ You only need to install it once &mdash; not every time you create a new Jekyll project.
```

Reply:
Ok, will update. Might be a little while as I just finished restoring from a Time Machine backup.

Tl;dr, 10.13.4 "delta" Installer is 💩 .

![smash-computer](https://m.popkey.co/cef55e/zgZ3R.gif)